packages/design-system/src/components/StepList/StepList.tsx

```diff
@@ -39,6 +39,10 @@ export interface StepListProps {
   substepsLabelText?: string;
 }
 
+/**
+ * For information about how and when to use this component,
+ * [refer to its full documentation page](https://design.cms.gov/components/step-list/).
```

Comment:
StepList has broken links in the prop table

Reply:
That's going to be dealt with in https://jira.cms.gov/browse/WNMGDS-2376 !